### ISSUE TITLE
updatedevices.pl: <<Make:PERL>>: bad interpreter: No such file or directory

### ISSUE THREAD

**Issue by diogoperdomo** [cvicente/Netdot#176]
Hi !
I´m trying to use CLI to discover devices using the command below:

``
./updatedevices.pl -B 192.168.1.0/24 -I -c public
``
``
bash: ./updatedevices.pl: <<Make:PERL>>: bad interpreter: No such file or directory
``
Does anyone know what's going on?

----------------------------- Netdor/bin/updatedevices.pl  ---------------------
```
#!<<Make:PERL>>
#
# updatedevices.pl - Update Netdot Devices
#
# This script retrieves SNMP information from one ore more devices
# and updates the Netdot database.  It can retrive basic device information
# as well as bridge forwarding tables and ARP caches. In addition, it can
# execute the topology discovery process.
#
use strict;
use lib "<<Make:LIB>>";
use Netdot::Model::Device;
use Netdot::Topology;
use Getopt::Long qw(:config no_ignore_case bundling);
use Log::Log4perl::Level;
...
```

**Comment by candlerb** [comment]
Yes: you didn't install Netdot. The installation process replaces these `<<placeholders>>` with correct values.

**Comment by diogoperdomo** [comment]
> Yes: you didn't install Netdot. The installation process replaces these `<<placeholders>>` with correct values.

Hi @candlerb . Thanks for your response!

I´m using a Docker (42wim/netdot) image to login with Admin user in web page.
I´m adding devices by especific IP and snmp version 2 public. The Netdot put all data in database very wel.

I think the NetDot is Installed atleast in WEB interface mode. 
The problem appear in CLI mode by accessing Docker Image  through terminals in linux. 
So, I gona check better the images inside the 42wim/netdot.
